src/scss/grommet-core/_objects.card.scss

```diff
@@ -50,14 +57,14 @@ $card-hover-color: #EBEBEB;
   //     }
   //   }
   //
-  //   .#{$grommet-namespace}control-icon-watch {
-  //     @include media-query(palm) {
-  //       // don't shrink video WatchIcon when in mobile
-  //       width: round($inuit-base-spacing-unit * 6);
-  //       height: round($inuit-base-spacing-unit * 6);
-  //     }
-  //   }
-  // }
+    .#{$grommet-namespace}control-icon-watch {
```

Comment:
Can we add this to the video icon in general? Perhaps a `responsive={false}` on the icons?